### ISSUE TITLE
Update for 1.12.1 

### ISSUE THREAD

**Issue by Greg-Hitchon** [pnp/generator-spfx#300]
#### General Information
* [ ] Usage
* [x] Development
* [ ] Documentation
* [ ] Feature Request

#### What Version you are running?
* NodeJS: 14.16.1
* NPM: 6.14.12
* @pnp/spfx: 1.16.1

#### Describe your problem
I get an error when running for 1.12.1 release "!!! Project needs to be upgraded !!!". Will there be an update to this generator, or is there a workaround?

**Comment by Greg-Hitchon** [comment]
Temporarily resolved this by updating the Microsoft generator and and  --skipVersionCheck flag, although you could just update the check.

**Comment by StfBauer** [comment]
It is not yet updated to 1.12.1 but will be updated to 1.12 in near future. Right now it just supports 1.11, not 1.12.1.

**Comment by Greg-Hitchon** [comment]
Yeah I get that. In our case we are using it to scaffold new customized webparts into an existing solution, so skipping the version check and overwrite of package.json works for us.

**Comment by araver** [comment]
Any updates on this?

**Comment by frankmatull** [comment]
Is it still planned to get updated to 1.12.1 or 1.13?

**Comment by StfBauer** [comment]
Yes - probably next year a new release comes out. Sadly it will be only managed by me and @chandaniprajapati atm.

Happy to review and except any PR from the community.
